EIPS/eip-draft-compact-sig.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 1
+title: EIP Compact Signature Representation
+status: Draft
+type: Informational
+author: Richard Moore <me@ricmoo.com>, Nick Johnson <nick@ethereum.org>
+created: 2019-03-14
+---
+
+## Simply Summary
+
+This proposal describes a compact representation of an Ethereum Signature.
+
+
+## Abstract
+
+The secp256k1 curve permits the computation of the public key of signed
+digest when coupled with a signature, which is used implicitly to
+establish the origin of an EOA-based transaction as well as on-chain in
+EVM contracts for example, in meta-transactions and multi-sig contracts.
+
+Currently signatures require 65 bytes to represent, which when aligned
+to 256-bit words, requires 96 bytes (with 31 zero bytes injected). With
+compact signatures, this can be reduced to 64 bytes, which remains 64
+bytes when word-aligned.
+
+
+## Motivation
+
+The motivations for a compact representation are to simplify handling
+transactions in client code, reduce gas costs and reduce transaction sizes.
+
+
+## Specification
+
+A secp256k1 signature is made up of 3 parameters, `r`, `s` and `v`. The `r`
+represents the `x` component on the curve (from which the `y` can be
+computed), and the `s` represents the challenge solution for signing by a
+private key. Due to the symmetric nature of an elliptic curve, a `v` is
+required, which indicates which of the 2 possible solutions was intended,
+by indicating its parity (odd-ness), 
+
+Two key observation are required to create a compact representation.
+
+First, the `v` parameter is always either 0 or 1 (canonically the values used
+have been 27 and 28, as these values didn't collide with other binary prefixes
+used in Bitcoin.)
+
+Second, the top bit of the `s` parameters is **always** 0, due to the use of
+canonical signatures which flip the solution parity to prevent negative values.
+
+So, we can use the top bit in the `s` parameter to store `v`.
+
+```python
```

Comment:
I'm usually not a fan of accepting code as specification. Do you want add a heading here along the lines "As an example this can be accomplished in Python with:" to signal this isn't a spec.